rust-bindings/bls-signatures/src/lib.rs

```diff
@@ -101,4 +101,20 @@ mod tests {
             grandchild_sk_u.g1_element().expect("cannot get public key")
         );
     }
+
+    #[test]
+    fn test_bls_public_key() {
+        println!("ody");
+
+        let seed = b"seedweedseedweedseedweedseedweed";
+        let scheme = LegacySchemeMPL::new();
+
+        let private_key = PrivateKey::key_gen(&scheme, seed).expect("unable to generate private key");
+        let public_key = private_key.g1_element().expect("unable to get public key");
+
+        println!("private_key:{:?}", private_key.to_bytes().as_slice());
```

Comment:
Can we find the value of `sk`, hardcode it and test the generated value against it? So far the test only validates that a value is generated but is not referenced against a known-good value.

Reply:
Agree. Should drop all println-s and instead compare the resulting privkey/pubkey to the expected one.